site/app/templates/grading/simple/Display.twig

```diff
@@ -1,20 +1,28 @@
 {% import _self as self %}
 
-<div class="content">
-    {#
-        Default is viewing your sections
-        Limited grader does not have "View All" option
-        If nothing to grade, Instructor will see all sections
-    #}
-    {% if show_all_sections_button %}
-        <div style="float: right; margin-bottom: 10px">
-            <a class="btn btn-default"
-               href="{{ core.buildUrl({'component': 'grading', 'page': 'simple', 'action': action, 'g_id': gradeable.getId(), 'sort': sort, 'view': view_all ? null : "all"}) }}">
-                {{ view_all ? "View Your Sections" : "View All" }}
-            </a>
-        </div>
-    {% endif %}
+<style>
+    .column-wrapper {
+        display:flex; 
+        justify-content: space-between;
+        align-items: flex-end;
+    }
+
+    .row-wrapper {
+        display:flex; 
+        flex-direction:column;
+    }
+    
+    #data-table td:focus, #data-table input:focus {
+        outline: 3px dashed #337ab7;
+    }
+
+    .fixed-top {
+        position: fixed;
+        top: 0;
+	}
+</style>
```

Comment:
We're trying to avoid inline styles going forward

Reply:
We're trying to phase out server.css so I figured keeping it on this page for now would make it easier to move foward. I mostly focused on having classes rather than the inline style attr. Should I make a new css page and link it?

Reply:
Moved to new css file